### PR TITLE
first step of 454

### PR DESCRIPTION
**Pull request by akasolace**
integrate power rating in DB
update/cleanup VEREIN table

1. changes proposed in this pull request:
 
   - fixes issue #__  
   
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)
 
 
2. changelog and release_notes ...

 - [ ] have been updated
 - [ ] do not require update

3. [Optional] suggested person to review this PR @___

### CODE REVIEW

File: src/main/java/core/model/misc/Verein.java [hunk 1]
```diff
@@ -76,6 +99,10 @@ public Verein(Properties properties) throws Exception {
         m_iFinancialDirectorLevels = DF.parse(properties.getProperty("financialdirectorlevels", "0")).intValue();
         m_iFormCoachLevels = DF.parse(properties.getProperty("formcoachlevels", "0")).intValue();
         m_iTacticalAssistantLevels = DF.parse(properties.getProperty("tacticalassistantlevels", "0")).intValue();
+        m_iGlobalRanking = DF.parse(properties.getProperty("globalranking", "0")).intValue();
+        m_iLeagueRanking = DF.parse(properties.getProperty("leagueranking", "0")).intValue();
+        m_iRegionRanking = DF.parse(properties.getProperty("regionranking", "0")).intValue();
+        m_iPowerRating = DF.parse(properties.getProperty("powerrating", "0")).intValue();
```
Comment (tychobrailleur) [inline src/main/java/core/model/misc/Verein.java:105]: If those properties come from the HRF file, there could be an issue here, as the keys seem camel case when converting XML to HRF:
https://github.com/akasolace/HO/pull/529/files#diff-f57a7f84ee6b7283a69deee3ea8d9764R362-R365

File: src/main/java/core/db/VereinTable.java [hunk 2]
```diff
@@ -41,24 +43,21 @@ protected void initColumns() {
...
+			//first delete existing entry
 			delete( awhereS, awhereV );
 
-			//insert vorbereiten
-			statement = "INSERT INTO "+getTableName()+" ( TWTrainer , COTrainer , Physiologen , Pschyologen , Finanzberater , PRManager , Aerzte , Jugend , Siege , Ungeschlagen , Fans , TacticAssist , FormAssist, HRF_ID ) VALUES(";
+			//create sql insertion statement
+			statement = "INSERT INTO "+getTableName()+" ( COTrainer , Pschyologen , Finanzberater , PRManager , Aerzte , Jugend , Siege , " +
+					"Ungeschlagen , Fans , GlobalRanking , LeagueRanking , RegionRanking , PowerRating , TacticAssist , FormAssist, HRF_ID ) VALUES(";
 			statement
 				+= (""
```
Comment (tychobrailleur) [inline src/main/java/core/db/VereinTable.java:58]: That first empty string could be removed too.

File: src/main/java/core/model/misc/Verein.java (hunk 1 above)
Comment (akasolace) [reply src/main/java/core/model/misc/Verein.java:105]: actually the get property calls toLowerCase (or something equivalent).  I tested it and it seems to work (values end up in database)

File: src/main/java/core/db/VereinTable.java (hunk 2 above)
Comment (akasolace) [reply src/main/java/core/db/VereinTable.java:58]: which empty string ?

**Pull request merged**